One test failed:
```
=================================== FAILURES ===================================
_________________ WhisperModelTest.test_equivalence_pt_to_flax _________________
[gw1] linux -- Python 3.8.12 /home/circleci/.pyenv/versions/3.8.12/bin/python

self = <tests.models.whisper.test_modeling_whisper.WhisperModelTest testMethod=test_equivalence_pt_to_flax>

    @is_pt_flax_cross_test
    def test_equivalence_pt_to_flax(self):
        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
        init_shape = (1,) + inputs_dict["input_features"].shape[1:]
    
        for model_class in self.all_model_classes:
            with self.subTest(model_class.__name__):
                fx_model_class_name = "Flax" + model_class.__name__
    
                if not hasattr(transformers, fx_model_class_name):
                    # no flax model exists for this class
                    return
    
                # Output all for aggressive testing
                config.output_hidden_states = True
                config.output_attentions = self.has_attentions
    
                fx_model_class = getattr(transformers, fx_model_class_name)
    
                # load PyTorch class
                pt_model = model_class(config).eval()
                # Flax models don't use the `use_cache` option and cache is not returned as a default.
                # So we disable `use_cache` here for PyTorch model.
                pt_model.config.use_cache = False
    
                # load Flax class
                fx_model = fx_model_class(config, input_shape=init_shape, dtype=jnp.float32)
    
                # make sure only flax inputs are forward that actually exist in function args
                fx_input_keys = inspect.signature(fx_model.__call__).parameters.keys()
    
                # prepare inputs
                pt_inputs = self._prepare_for_class(inputs_dict, model_class)
    
                # remove function args that don't exist in Flax
                pt_inputs = {k: v for k, v in pt_inputs.items() if k in fx_input_keys}
    
                # send pytorch inputs to the correct device
                pt_inputs = {
                    k: v.to(device=torch_device) if isinstance(v, torch.Tensor) else v for k, v in pt_inputs.items()
                }
    
                # convert inputs to Flax
                fx_inputs = {k: np.array(v) for k, v in pt_inputs.items() if torch.is_tensor(v)}
    
                fx_state = convert_pytorch_state_dict_to_flax(pt_model.state_dict(), fx_model)
                fx_model.params = fx_state
    
                # send pytorch model to the correct device
                pt_model.to(torch_device)
    
                with torch.no_grad():
                    pt_outputs = pt_model(**pt_inputs)
                fx_outputs = fx_model(**fx_inputs)
    
                fx_keys = tuple([k for k, v in fx_outputs.items() if v is not None])
                pt_keys = tuple([k for k, v in pt_outputs.items() if v is not None])
    
                self.assertEqual(fx_keys, pt_keys)
>               self.check_pt_flax_outputs(fx_outputs, pt_outputs, model_class)

tests/models/whisper/test_modeling_whisper.py:865: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_modeling_common.py:2098: in check_pt_flax_outputs
    self.check_pt_flax_outputs(
tests/test_modeling_common.py:2123: in check_pt_flax_outputs
    self.check_pt_flax_outputs(fx_output, pt_output, model_class, tol=tol, name=attr)
tests/test_modeling_common.py:2152: in check_pt_flax_outputs
    self.assertLessEqual(
E   AssertionError: 1.1086464e-05 not less than or equal to 1e-05 : outputs.encoder_last_hidden_state: Difference between PyTorch and Flax is 1.1086463928222656e-05 (>= 1e-05).
```
But I doubt that it is due to the changes added. @sgugger correct me if I am wrong

--- a/src/transformers/modeling_utils.py
+++ b/src/transformers/modeling_utils.py
@@ -378,7 +378,7 @@ def load_sharded_checkpoint(model, folder, strict=True, prefer_safe=False):
                 load_safe = True  # load safe due to preference
             else:
                 logger.warning(
-                    f"Cannot load sharded checkpoint at {folder} safely since safetensors are not installed!"
+                    f"Cannot load sharded checkpoint at {folder} safely since safetensors is not installed!"
                 )
         elif not index_present:
             load_safe = True  # load safe since we have no other choice

--- a/src/transformers/training_args.py
+++ b/src/transformers/training_args.py
@@ -262,7 +262,7 @@ class TrainingArguments:
         save_total_limit (`int`, *optional*):
             If a value is passed, will limit the total amount of checkpoints. Deletes the older checkpoints in
             `output_dir`.
-        save_safetensors (`bool`, *optional*):
+        save_safetensors (`bool`, *optional*, defaults to `False`):
             Use [safetensors](https://huggingface.co/docs/safetensors) saving and loading for state dicts instead of
             default `torch.load` and `torch.save`.
         save_on_each_node (`bool`, *optional*, defaults to `False`):

--- a/src/transformers/training_args.py
+++ b/src/transformers/training_args.py
@@ -264,8 +264,7 @@ class TrainingArguments:
             `output_dir`.
         save_safetensors (`bool`, *optional*):
             Use [safetensors](https://huggingface.co/docs/safetensors) saving and loading for state dicts instead of
-            default `torch.load` and `torch.save`. This will default to True if safetensors are installed and False
-            otherwise. To enforce using torch functions, set this to False.
+            default `torch.load` and `torch.save`.
         save_on_each_node (`bool`, *optional*, defaults to `False`):
             When doing multi-node distributed training, whether to save models and checkpoints on each node, or only on
             the main one.

